charts/renku/requirements.yaml

```diff
@@ -18,3 +18,7 @@ dependencies:
 - name: postgresql
   version: 0.14.4
   repository: "@stable"
+- name: renku-gateway
+  alias: gateway
+  version: "0.1.0-97879d4"
+  repository: "https://swissdatasciencecenter.github.io/helm-charts/"
```

Comment:
please include the gateway chart in the html template so they appear on https://swissdatasciencecenter.github.io/helm-charts/

Reply:
Done!